### ISSUE TITLE
create legacy lessons section

### ISSUE THREAD

**Issue by ohasselblad** [CartoDB/academy#338]

**Comment by ohasselblad** [comment]
@carlostallon this would be a new lesson card that would not be as prominent

**Comment by arianaescobar** [comment]
@ohasselblad do we still want this for the redesign (v1)?

**Comment by ohasselblad** [comment]
Yes, please! I'm realizing now that it was not included in the IA -- sorry.

**Comment by arianaescobar** [comment]
These should be on the academy index or only inside their specific subjects?

**Comment by ohasselblad** [comment]
academy index -- hopefully release of all of this coincides with builder release to the public

**Comment by arianaescobar** [comment]
and as another subject named "Legacy lessons" or something like that?

**Comment by ohasselblad** [comment]
Yes, exactly. We were thinking it would be good to have it appear differently.. like having different styles, being physically separate (bottom of page), etc.
